apps/omg/lib/omg/signature.ex

```diff
@@ -42,6 +42,9 @@ defmodule OMG.Signature do
   This implements Eq.(208) of the Yellow Paper, adapted from https://stackoverflow.com/a/20000007
 
   """
+  def recover_public(hash, <<r::integer-size(256), s::integer-size(256), v::integer-size(8)>>, chain_id \\ nil),
+    do: recover_public(hash, v, r, s, chain_id)
```

Comment:
I would be in favor of going multi-lines when it doesn't fit on one line, and enforce it as a styling guideline.

```suggestion
def recover_public(hash, <<r::integer-size(256), s::integer-size(256), v::integer-size(8)>>, chain_id \\ nil) do
  recover_public(hash, v, r, s, chain_id)
end
```

Reply:
Hmm, don't think you can use my suggestion above to create the commit, lines are wrong (with multi-lines selection).